styles/diver.css

```diff
@@ -0,0 +1,97 @@
+.diver {
+    background: 
+    url("../images/diver_bubble.png") 100% no-repeat,
+    url("../images/diver.png") 100% no-repeat,
+    url("../images/scuba.png") 100% no-repeat,
+    url("../images/diver_bubls.png") 100% no-repeat,
+    url("../images/diver_bubl.png") 100% no-repeat,
+    linear-gradient(178.68deg, #4F3963 -2.24%, #764469 31.33%, #5E3E66 76.25%, #2C2142 94.12%);
+    background-position: 
+    top 354px left 100px,
+    top 111px right 106px,
+    top 1326px left 94px,
+    top 1495px right 90px,
+    top 1065px right 779px,
+    top 0 left 0;
+
+    padding-top: 460px;
+    padding-bottom: 70px;
+}
+
+.diver__cards {
+    display: flex;
+    flex-wrap: wrap;
+    column-gap: 44px;
+}
+
+.diver__card {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    overflow: hidden;
+    position: relative;
+    flex-basis: calc(50% - 22px);
+
+    height: 308px;
+
+    box-shadow:
+    linear-gradient(178.68deg, #4F3963 -2.24%, #764469 31.33%, #5E3E66 76.25%, #2C2142 94.12%);
+    border-radius: 5px;
+    background-color: #523A64;
+}
+
+.diver__card-terminology {
+    font-size: 24px;
+    line-height: 28px;
+    color: #F2F2F2;
+    font-weight: 700;
+
+    padding: 87px 80px 25px 125px;
+}
+
+.diver__card-description {
+    font-size: 18px;
+    font-weight: 400;
+    color: #F2F2F2;
+
+    padding-left: 125px;
+    padding-right: 80px;
+}
+
+.diver__card:nth-child(2) {
+    margin-top: 232px;
+}
+
+.diver__card:nth-child(even) {
+    margin-bottom: 160px;
+}
+
+.diver__card:nth-child(2n+3) {
+    margin-top: -75px;
```

Comment:
![image](https://user-images.githubusercontent.com/34751890/178228340-2517fa22-50cb-4465-af98-5c7671cdff43.png)
```suggestion
    margin-top: -76px;
```